configs/namespaces_syscall.go

```diff
@@ -4,17 +4,22 @@ package configs
 
 import "syscall"
 
+var (
+	CLONE_SECCOMP = 0x10000 //diffrent from other flag, hard code
```

Comment:
CLONE_NEWUSER                    = 0x10000000
CLONE_NEWSECCOMP is a pseudo flag, which is diffrent other CLONE_xx flag, and it can not pass to clone.